taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -313,6 +314,59 @@ def create_report(self):
                 with_log_files=(logprefix is not None),
                 prefix="{}_aggregate".format(logprefix))
 
+    def copy_report(self):
+        """ Copies the aggregate report and version reports files to a specified outbox directory.
+            :returns: list of the paths to the files it has successfully copied (i.e. the targets)
+        """
+
+        def find_from_files_to_deliver(pattern):
+            """ Searches the nested list of `files_to_deliver` for files matching the provided pattern
+                :param pattern: the regex pattern to search for
+                :returns: single matching file
+                :raises: AssertionError if there is not strictly one match for the pattern
+            """
+
+            matches = []
+
+            for file_list in self.files_to_deliver:
+                for f in file_list:
```

Comment:
I think I'm happy with this for now, though you are of course right. However - this might be more robust to changes...